code/script.js

```diff
@@ -1,51 +1,316 @@
+let developerOption = "";
+
+
+// -----------------------------------------------------
+// functions
+
+// 0. Introduction
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
   if (sender === 'user') {
+    // console.log("a user message")
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="code/assets/user.png" alt="User" />  
       </section>
     `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
   } else if (sender === 'bot') {
+    // console.log("a bot message")
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="code/assets/bot.png" alt="Bot" />
         <div class="bubble bot-bubble">
-          <p>${message}</p>
+          <p id="message">${message}</p>
         </div>
       </section>
     `
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
+  chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  // here we call the function showMessage
+  showMessage("Welcome to dreamhoster.net, your service provider for professional hosting. What services are you interested in?", 'bot');
+}
+
+// 1a. all general service offers
+const serviceOffers = () => {
+  showMessage(`&bull; Domain registration<br>&bull; Web site hosting<br>&bull; Domain registration and hosting`, "bot");
+
+  // create 3 buttons
+  document.getElementById("input-wrapper").innerHTML = `
+  <div id="services-wrapper" style="display: flex; justify-content: space-evenly">
+    <button id="domain" value="domain">Domain<br>registration:<br>15 credits / year</button>
+    <button id="hosting" value="hosting">Hosting:<br>5  credits / month</button>
+    <button id="domainAndHosting" value="domainAndHosting">Registration<br>and Hosting:<br>15% discount</button>
+  </div>`;
+  // additional div for targeting content and serving as flex container, input-wrapper needs to ramain. css-styling only possible here inline, not in css file.
+  // do not nest ""
+
+  const servicesWrapper = document.getElementById("services-wrapper");
+
+  servicesWrapper.addEventListener("click", function (event) {
```

Comment:
your comments are really helpful and it shows how much you have thought about the flow of the bot, I'm sure your style of commenting will be helpful for you in weeks to come looking back on these initial projects